trunk/SUAVE/Core/DataOrdered.py

```diff
@@ -764,6 +764,8 @@ def __str2(self,indent=''):
                         val = value.__str__(indent+new_indent)
                     except RuntimeError: # recursion limit
                         val = ''
+                    except:
+                        val = value.__str__(indent+new_indent)                                    

```

Comment:
This just runs the try statement's line again (in the other file the try statement used __str2 but here it uses __str__)

Reply:
Thanks for pointing this out. I've fixed it.